RadixWallet/Features/Signing/Coordinator/Signing+View.swift

```diff
@@ -13,41 +13,7 @@ extension Signing {
 		}
 
 		public var body: some SwiftUI.View {
-			SwitchStore(store.scope(state: \.step, action: Action.child)) { state in
-				switch state {
-				case .signWithDeviceFactors:
-					CaseLet(
-						/Signing.State.Step.signWithDeviceFactors,
-						action: Signing.ChildAction.signWithDeviceFactors,
-						then: { SignWithFactorSourcesOfKindDevice.View(store: $0) }
-					)
-
-				case .signWithLedgerFactors:
-					CaseLet(
-						/Signing.State.Step.signWithLedgerFactors,
-						action: Signing.ChildAction.signWithLedgerFactors,
-						then: { SignWithFactorSourcesOfKindLedger.View(store: $0) }
-					)
-				}
-			}
-		}
-	}
-
-	/// A wrapper for Signing.View to be used in a sheet context
-	@MainActor
-	public struct SheetView: SwiftUI.View {
-		private let store: StoreOf<Signing>
-
-		public init(store: StoreOf<Signing>) {
-			self.store = store
-		}
-
-		public var body: some SwiftUI.View {
-			WithNavigationBar {
-				store.send(.view(.closeButtonTapped))
-			} content: {
-				View(store: store)
-			}
+			SignWithFactorSource.View(store: store.scope(state: \.signWithFactorSource, action: \.child.signWithFactorSource))
```

Comment:
One could argue that we could merge `Signing` with `SignWithFactorSource`. However, I would rather keep it separately so that each of them is in charge of one task:
- `SignWithFactorSource`: responsible for the signature with the given factor source
- `Signing`: responsible for managing all the factor sources and acting accordingly when all signatures have been collected.

Reply:
Yes we cannot merge those, the `Signing` is the coordinator and will need to be kept around.